app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,26 @@
+class MoviesController < ApplicationController
+  include MoviesHelper
+  include ApplicationHelper
+  require 'open-uri'
+
+  def index
+    find_recent_movies
+    find_opening_movies
+    find_best_movies
+    find_top_box
```

Comment:
👍 descriptive method names allow for extracting the logic into the helper, makes it much easier to read your controller